========================================
            ERROR EN EL PROGRAMA
========================================
Hora del error: Thu Aug 14 23:55:49 2025
Tipo de error: TypeError
Mensaje: IAA.move() missing 3 required positional arguments: 'ball_x', 'ball_speed_x', and 'ball_speed_y'

--- Traceback ---
Traceback (most recent call last):
  File "C:\Users\Wilson\.vscode\ai\run_game.py", line 573, in <module>
    start_diablo_mode(ia_players)
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "C:\Users\Wilson\.vscode\ai\run_game.py", line 508, in start_diablo_mode
    winner, _, _ = start_game(player1, player2, player1_name, player2_name,
                   ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              time_limit=current_time_limit, consecutive_score_limit=1, point_score_limit=1,
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              diablo_mode=True, diablo_round=round_num, diablo_points=diablo_points, diablo_victories=diablo_victories,
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                              ball_speed=current_ball_speed)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Wilson\.vscode\ai\run_game.py", line 124, in start_game
    player1_y += player1.move(balls[0]['y'], player1_y)
                 ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: IAA.move() missing 3 required positional arguments: 'ball_x', 'ball_speed_x', and 'ball_speed_y'
